src/transformers/models/convnext/modeling_tf_convnext.py

```diff
@@ -445,6 +433,7 @@ def __init__(self, config, *inputs, add_pooling_layer=True, **kwargs):
 
     @add_start_docstrings_to_model_forward(CONVNEXT_INPUTS_DOCSTRING)
     @replace_return_docstrings(output_type=TFBaseModelOutputWithPooling, config_class=_CONFIG_FOR_DOC)
+    @unpack_inputs
```

Comment:
Sorry to be a pain about this -- in the other PRs, the new decorator is the top decorator. For consistency, can you please move it so as to be the top decorator? 🤗  (there is another instance of this in these changes)